cookbook/add-tabs.rst

```diff
@@ -0,0 +1,300 @@
+Adding tabs to Sulu admin interface
+===================================
+
+.. note:: It is recommended to read `Extend admin Ui`_ beforehand to get a better understandig of how Sulu admin classes work.
+
+
+This tutorial will walk you through the process of adding extra tabs to the user interface of Sulu.
+
+This could be useful in a few situations, a few examples below:
+
+- Saving extra data for pages
+- Saving extra data in webspaces
+- Saving extra data for users
+
+In this example we'll be adding a "Socials" tab to the PageAdmin of Sulu.
+
+.. figure:: ../img/socials-tab.png
+
+Create the form for this view
+-----------------------------
+You have to create the form that is rendered in this view. For this we create an XML file like your used to.
+Create your form: ``config/forms/page_socials.xml``. Notice how we use slashes in the names of the properties, this will be of later use.
+
+.. code-block:: xml
+
+    <?xml version="1.0" ?>
+    <form xmlns="http://schemas.sulu.io/template/template"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/form-1.0.xsd"
+    >
+        <key>page_socials</key>
+
+        <properties>
+            <section name="twitter">
+                <meta>
+                    <title>Twitter</title>
```

Comment:
This should cause warnings in the UI, because if you don't pass a language to the `title` tag it will be tried to be used as a translation key, and this translation key does not exist. We could work around that by writing it the following way instead:

```xml
<title lang="en">Twitter</title>
```